gemelli/optspace.py

```diff
@@ -374,10 +374,8 @@ def joint_solve(self, multiple_obs):
             S_shared = np.diag(S_shared)
             S_shared = S_shared / np.linalg.norm(S_shared)
             U_shared = np.average(sample_loadings, axis=0)
-            U_shared -= U_shared.mean(0)
```

Comment:
This seems like an impactful change. I'm not sure though if it is appropriate to couple to a unit test? I'll defer on that

Reply:
I would but it is hard to add a test for since it is so deep.